docs/manage/fleet/robots.md

```diff
@@ -111,6 +111,6 @@ Click on the **Generate Key** button to generate a new key.
 
 ## Delete a robot
 
-You can delete a robot by checking the **Sure?** box in the lower left of the robot page and clicking **Delete robot**.
+You can delete a robot by navigating to its **Config** page in [the Viam app](https://app.viam.com) and selecting **Sure?** and **Delete robot** in the lower left corner of the page.
```

Comment:
```suggestion
You can delete a robot by navigating to its page in [the Viam app](https://app.viam.com) and selecting **Sure?** and **Delete robot** in the lower left corner of the page.
```
You can do this from any of the tabs, not just from the config tab.

Reply:
@JessamyT True!! However, you cannot do this from setup tab, which is where they land when opening the robot's page. I figured `navigate to config tab` would be simpler than saying Config, Security, etc-- is that ok with you/does that make sense?

Reply:
Actually I saw it on the setup tab 10 minutes ago, but right now it looks like there's an app bug such that you can't do it from the config tab! The scrolling seems to be bugging. So I think you can actually do it from setup but I can't prove it right now because bug :D

Reply:
I've filed an app bug ticket

Reply:
Turns out ticket was a duplicate :D but did confirm with eng that there should be a delete robot box on the setup tab.

Reply:
Ok, cool. Would you recommend I just close this pr, or should we merge it for now and update once bug is closed? @JessamyT up to you